src/pages/mesh/advanced/caching/index.md

```diff
@@ -147,7 +156,23 @@ Example 2
 
   - no-store
 
-### In the mesh configuration file
+Example 3
+
+Public and private headers are mutually exclusive, since `private` is more restrictive, API Mesh selects the values associated with the `private` header.
```

Comment:
```suggestion
Public and private headers are mutually exclusive. Since `private` is more restrictive, API Mesh selects the values associated with the `private` header.
```